src/styles/theme/oc-text.scss

```diff
@@ -1,12 +1,5 @@
-body,
-h1,
-h2,
-h3,
-h4,
-h5,
-h6,
-p {
-  color: var(--oc-color-text-default);
+html * {
+  /*color: var(--oc-color-text-default);*/
```

Comment:
Hm... What about keeping the old css for the color, and just adding the `html * {}` for the font? Don't we loose text colors with your current changeset?

Reply:
it doesnt seem like anything got broken

Reply:
Just checked it out locally, rebased on master and used in it in web with a local link. Headings stay black independently of the picked theme, so we need some sort of awareness of the text color variable here. Let me check which approach works best

Reply:
Setting the color on the HTML tag messes with quite some other components, so I'd suggest we keep the coloring as it was before and only add the ` html * { font-family }` stuff :)

Reply:
alright will do that